idaes/gas_solid_contactors/unit_models/moving_bed.py

```diff
@@ -620,9 +647,15 @@ def solid_super_vel(b, t, x):
                              doc="Gas side pressure drop calculation -"
                                  "simplified pressure drop")
             def gas_phase_config_pressure_drop(b, t, x):
-                return b.gas_phase.deltaP[t, x]*1e5 == -0.2*(
+                #  add 1/pyunits.s to account for missing parameter units
+                return pyunits.convert(
+                    b.gas_phase.deltaP[t, x],
+                    to_units=units_meta_solid('pressure') /
+                    units_meta_solid('length')) == -0.2/pyunits.s*(
```

Comment:
What is the `-0.2` in this case? Is it a unitted constant (i.e. it has units of `1/s`?

Reply:
Yes, this is a unitted constant to satisfy unit consistency.

Reply:
I would change the comment to note that 0.2/s is a unitted costant, and proably put the actual parameter in parenthesis for clarity: `(0.2/pyunits.s)`